airflow/www/security.py

```diff
@@ -453,12 +453,19 @@ def merge_pv(perm, view_menu):
 
     def update_admin_perm_view(self):
         """
-        Admin should have all the permission-views.
+        Admin should have all the permission-views, except the dag views.
+        because Admin have already have all_dags permission.
```

Comment:
Typo `have already have` here.
Maybe you meant to type `Admin already has`

Reply:
Yes, I should modify here.